Pages/OverlayPage.js

```diff
@@ -6,6 +6,9 @@ var OverlayPage = function() {
     var overlaySelf = $locator('.overlay-media');
     var overlayContentLink = $locator('.overlayLeftPanel .content-title-link');
     var overlayTwitterShare = $locator('.overlayLeftPanel .btn.share-twitter');
+    var TweetAuthorLeft = $locatorxpath("//span[contains(@class,'overlay-author')]");
+    var FirstReTweetAuthorName = $locatorxpath("(//div[@class='overlayRightPanel']//blockquote//span[@class='username']/a)[1]");
```

Comment:
Why do we need a first retweet authorname. Replace this with a function that accepts a parameter like index and returns a locator.